src/AFQMC/Estimators/EstimatorHandler.h

```diff
@@ -136,6 +137,9 @@ class EstimatorHandler: public AFQMCInfo
           } else if (name == "back_propagation") {
             estimators.emplace_back(static_cast<EstimPtr>(std::make_shared<BackPropagatedEstimator>(TGgen.getTG(1),info,title,cur,walker_type,wset,*wfn,prop0,impsamp)));
             hdf_output = true;
+          } else if (name == "new_back_propagation") {
+            estimators.emplace_back(static_cast<EstimPtr>(std::make_shared<BackPropagatedEstimator_>(TGgen.getTG(1),info,title,cur,walker_type,wset,*wfn,prop0,impsamp)));
```

Comment:
Is this a temporary thing? Will all estimators go through the new handler eventually?

Reply:
Just a comment: I've observed that use of the word "new" when introducing an updated version of a feature becomes outdated very quickly in retrospect (i.e. usually the "new" thing becomes standard and the original is eventually discarded).

Reply:
Agree. These litter the source. Call the new thing by the new algorithm; if it is a replacement for the old algo delete the old implementation.

Reply:
I left the old one for comparison and forgot to remove it/rename the new one.
Fixed it.